dev-docs/prebid-1.0-API.md

```diff
@@ -65,21 +61,38 @@ The input to `pbjs.setConfig` must be JSON (no JavaScript functions are allowed)
 {% highlight js %}
 
 pbjs.setConfig({
-    "currency": {
-        // Enables currency feature -- loads the rate file
+    currency: {
         "adServerCurrency": "JPY",
-        // Allows the publisher to override the default rate file
-        "conversionRateFile": "url"
+        "conversionRateFile": "<url>"
     },
-    "debug": true, // Previously `logging`
-    "s2sConfig": { ... },
-    "priceGranularity": "medium",
-    "enableSendAllBids": false, // Default will be `true` as of 1.0
-    "bidderSequence": "random",
-    "bidderTimeout": 700, // Default for all requests.
-    "publisherDomain": "abc.com", // Used for SafeFrame creative.
-    "pageOptions": { ... },
-    "sizeConfig": { ... }
+    debug: true, // Previously `logging`
+    s2sConfig: { ... },
+    priceGranularit": "medium",
+    enableSendAllBids: false, // Default will be `true` as of 1.0
+    bidderSequence: "random", // Default is random
+    bidderTimeout: 700, // Default for all requests.
+    publisherDomain: "abc.com", // Used for SafeFrame creative.
+    pageOptions: { ... },
+    sizeConfig: { ... },
+    cache: {url: "<prebid cache url>"}
+});
+
+{% endhighlight %}
+
+## No More Default Endpoints
```

Comment:
Does this mean the following no longer works as of [the 1.0 release](https://github.com/prebid/Prebid.js/releases/tag/1.0.0)?

```
pbjs.setConfig({
        usePrebidCache: true
});
```

If so there are a lot of video docs that need updating as a result of this change.  Looks like this is a result of <https://github.com/prebid/Prebid.js/pull/1904> yes?

Reply:
Captured in <https://github.com/prebid/prebid.github.io/issues/482>

Reply:
Right - now it would to be:
```
pbjs.setConfig({
        usePrebidCache: true,
        cache: {url: ""}
});
```